source/index.md

```diff
@@ -11,11 +13,57 @@ build Ember apps, focusing on the most widely-used features of Ember.js.
 For exhaustive documentation of every Ember feature and API, see the
 [Ember.js API documentation](http://emberjs.com/api/).
 
-These guides are written in Markdown and are
-[available on GitHub](https://github.com/emberjs/guides/).
-If there is something missing, or you find a typo or
-mistake, please help us by filing an issue or submitting a pull
-request. You can easily jump to Github's editor for any page by
-clicking the pencil icon in the upper right corner.
+## Assumptions
+
+While we try to make the guides as beginner-friendly as we can, we must
+establish a baseline so that the guides can keep focused on Ember.js
+functionality. We will try to link to appropriate documentation whenever
+a concept is introduced.
+
+To make the most out of the guides, you should have a working knowledge of:
+
+* **HTML, CSS, JavaScript** - the building blocks of web pages. You can find documentation of each of these technologies at the [Mozilla Developer Network][mdn].
+* **Promises** - the native way to deal with asynchrony in your JavaScript code. See the relevant [Mozilla Developer Network][promises] section.
+* **ES2015 modules** - you will better understand Ember CLI's project structure and import paths if you are comfortable with [ES6 JavaScript Modules][js-modules].
```

Comment:
For reference, the JSmodules site runs from this repo: https://github.com/wycats/jsmodules